### PR TITLE
A first set of classes common for both ITS and MFT

### PR DESCRIPTION
**Pull request by iouribelikov**

### CODE REVIEW

File: Detectors/ITSMFT/ITS/base/include/ITSBase/GeometryTGeo.h [hunk 1]
```diff
@@ -16,13 +16,20 @@ class TGeoPNEntry; // lines 17-17
 
 namespace AliceO2
 {
+  namespace ITSMFT
+  {
+    class Segmentation;
+  }
+}
+using AliceO2::ITSMFT::Segmentation;
```
Comment (ktf) [inline Detectors/ITSMFT/ITS/base/include/ITSBase/GeometryTGeo.h]: `using` statements in headers are forbidden by our policy (and for a good reason, IMHO).

File: Detectors/ITSMFT/ITS/reconstruction/include/ITSReconstruction/TrivialClusterer.h [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#include "Rtypes.h"  // for TrivialClusterer::Class, Double_t, ClassDef, etc
+
+class TClonesArray;
+
+namespace AliceO2 {
+  namespace ITSMFT {
+    class SegmentationPixel;
+  }
+}
+using AliceO2::ITSMFT::SegmentationPixel;
```
Comment (ktf) [inline Detectors/ITSMFT/ITS/reconstruction/include/ITSReconstruction/TrivialClusterer.h]: `using` statements are forbidden in headers.

File: Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/DigitStave.h [hunk 3]
```diff
@@ -15,9 +15,15 @@
 class TClonesArray;
 
 namespace AliceO2 {
-namespace ITS {
+   namespace ITSMFT {
+      class Digit;
+   }
+}
+
+using AliceO2::ITSMFT::Digit;
```
Comment (ktf) [inline Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/DigitStave.h]: `using` statements in headers are forbidden by our policy.

**Comment by iouribelikov** [comment]
Thank you, Giulio, for pointing this out.  
Indeed, there were many places like that !
All of them have been fixed now.
Cheers,
Iouri.

**Pull request merged**